packages/orchestrator-ui-components/src/components/WFOForms/AutoFieldLoader.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+    BoolField,
+    DividerField,
+    LabelField,
+    LongTextField,
+    NumField,
+    TextField,
+} from './formFields';
+import { Context, GuaranteedProps } from 'uniforms';
+import { AutoField } from 'uniforms-unstyled';
+
+export function autoFieldFunction(
+    props: GuaranteedProps<unknown> & Record<string, any>,
+    uniforms: Context<unknown>,
+) {
+    const { fieldType, field } = props;
+    const { format } = field;
+
+    switch (fieldType) {
+        case Number:
+            switch (format) {
```

Comment:
a switch in a switch migt be better to be moved to a separate method.
case Number: getNumberFormat(format) -- containing the other switch

Reply:
Copied from V1; the actual swithc will be a lot bigger when we add all Fields